### ISSUE TITLE
model.fit(); and console.log(response.history.loss[0]);

### ISSUE THREAD

**Issue by JAGUAR1617** [CodingTrain/website#1992]
i m not getting anything in console after using model.fit() please help me out
model.fit(xs, ys).then((History) => console.log(History.history));

**Comment by JAGUAR1617** [comment]
done small mistake
